modules/todoListActivities.test.js

```diff
@@ -0,0 +1,146 @@
+import AvailableActivities from './todoListActivities.js';
+
+describe('AvailableActivities', () => {
+  describe('addActivity', () => {
+    describe('addActivity', () => {
+      beforeEach(() => {
+        global.localStorage = {
+          getItem: jest.fn(),
+          setItem: jest.fn(),
+          clear: jest.fn(),
+        };
+        localStorage.clear();
+      });
+
+      // Test 1
+      test('should add a new activity to the activities array and update the DOM', () => {
+        const listElement = document.createElement('ul');
+        const activities = new AvailableActivities();
+        const activity = {
+          description: 'New Activity',
+          completed: false,
+          index: 1,
+        };
+
+        const setItemMock = jest.spyOn(Storage.prototype, 'setItem');
+        const result = activities.addActivity(activity, listElement);
+
+        expect(result).toHaveLength(1);
+        expect(result[0]).toEqual(activity);
+        expect(listElement.children).toHaveLength(1);
+        expect(listElement.children[0].textContent).toBe(
+          'New Activity',
+        );
+        expect(setItemMock).toHaveBeenCalledWith(
+          'savedActivities',
+          JSON.stringify(result),
+        );
+      });
+
+      // Test 2
+      test('should not add an activity with an empty description and not update the DOM', () => {
+        const listElement = document.createElement('ul');
+        const activities = new AvailableActivities();
+        const activity = {
+          description: '',
+          completed: false,
+          index: 1,
+        };
+
+        const setItemMock = jest.spyOn(localStorage, 'setItem');
+        const result = activities.addActivity(activity, listElement);
```

Comment:
- [ ] You did a great job so far Unfortunately, these 2 test cases are failing (see the image below). Please make sure that all test cases are passing successfully.

![image](https://github.com/rmiyoyo/Testing-Todo-List/assets/68796232/fee66b8c-ff82-492a-a9dd-cdb7671ad3fa)

[Hint] These 2 tests are failing because of ` global.localStorage`, It's supposed to be `localStorage` only, But then you will have a linters issue with this so find a workaround to solve the issue.

Reply:
The tests pass on my computer, how should I go about this?